Бортовые номера

kosmik641 opened the pull request:
resolve #555 
@SmileDoge решил сделать всё у себя в форке, я не смог внести правки

kosmik641 committed:
Бортовые номера
Модели циферек
Расстоновка моделей на всех составах

ImPerzz reviewed (changes requested):
Требуется исправить цифру 1, у неё не должен быть такой же отступ как у других
<img width="1468" height="771" alt="image" src="https://github.com/user-attachments/assets/3a58196f-420b-489e-aec9-b85f52d2ee8f" />

HellReach committed:
fix: resolve issue preventing players from sitting in seats

kosmik641 committed:
Announcer.
Добавлена отсутствующая система Announcer на 81-721
Добавлена проверка наличия системы Announcer

kosmik641 committed:
81-717 ЛВЗ.
Добавлено задание давления в главном резервуаре

kosmik641 committed:
Спавнер.
Фикс ошибки при смене оружия. self.GhostEntities удаляется самим GMod'ом

kosmik641 committed:
Бортовые номера.
Установка моделей цифр бортовых номеров в modelcallback из-за бага, когда моделька улетает в центр вагона.

kosmik641 commented:
Будем принимать пакетно из `unstable` в `dev`
Добавлено в [8a6855f](https://github.com/metrostroi-repo/MetrostroiAddon/commit/8a6855f35b3336d0cec7bbd720c935fcd0f1814f)
